docs/installation.md

```diff
@@ -73,7 +73,11 @@ scenarios like CIs. Many thanks to [GoDownloader][godownloader] for allowing
 easily generating this script.
 
 ```bash
-curl -sL https://taskfile.dev/install.sh | sh
+# For Default Installion to ./bin with debug logging
+url -sL https://taskfile.dev/install.sh | sh -s -- -d 
```

Comment:
What's this `url` command? It isn't available on macOS on ZSH. I'd personally keep `curl` it's its way more commonly available in the wild.

Reply:
This `-d` flag errors for me, on `curl` at least

Reply:
So I just checked in container and `curl -sL https://taskfile.dev/install.sh | sh -s -- -d` worked without any error. I thought the `--` designates the end of shell argument parsing and `-d` should be passed to the executable task then, right?

If so `-d` should work on macOS and Linux as it is the argument for debug output for your task executable right?

Regarding the install path. I'm still a bit fuzzy on where `/opt/usr` vs `/usr/local/bin` and other paths are the best fit as my background is Windows primarily. To my understanding global user tools would fit either in the `/opt` or `/usr/local/bin` , but maybe there is a better practice you can let me know of.

Reply:
@andreynering  try this out now. I just tested on macOS and Linux and it works fine now for me.